awscfn/table_awscfn_mapping.go

```diff
@@ -0,0 +1,150 @@
+package awscfn
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"gopkg.in/yaml.v3"
+)
+
+func tableAWSCFNMapping(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "awscfn_mapping",
+		Description: "Cloudformation mapping information.",
+		List: &plugin.ListConfig{
+			Hydrate:    listAWSCloudFormationMappings,
+			KeyColumns: plugin.OptionalColumns([]string{"path"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "Parameter name.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "key",
+				Description: "The data type for the parameter.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "value",
+				Description: "An array containing the list of values allowed for the parameter.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "start_line",
+				Description: "Starting line number.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "path",
+				Description: "Path to the file.",
+				Type:        proto.ColumnType_STRING,
+			},
+		},
+	}
+}
+
+type awsCFNMapping struct {
+	Name      string
+	Key       string
+	Value     interface{}
+	StartLine int
+	Path      string
+}
+
+type MappingsStruct struct {
+	Mappings  map[string]interface{} `cty:"Mappings"`
+	Resources map[string]interface{} `cty:"Resources"`
+}
+
+func listAWSCloudFormationMappings(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// #1 - Path via qual
+	// If the path was requested through qualifier then match it exactly. Globs
+	// are not supported in this context since the output value for the column
+	// will never match the requested value.
+	//
+	// #2 - Path via glob paths in config
+	var paths []string
+	if d.KeyColumnQuals["path"] != nil {
+		paths = []string{d.KeyColumnQuals["path"].GetStringValue()}
+	} else {
+		var err error
+		paths, err = listFilesByPath(ctx, d.Connection)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	for _, path := range paths {
+		// Read files
+		content, err := ioutil.ReadFile(path)
+		if err != nil {
+			plugin.Logger(ctx).Error("awscfn_mapping.listAWSCloudFormationMappings", "file_error", err, "path", path)
+			return nil, fmt.Errorf("failed to read file %s: %v", path, err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("failed to read file %s: %w", path, err)
```

While working with @LalitTurbot on the Snowflake plugin, I think we ran into a question around how should error messages be formatted, and from there, I believe we found that `%w` is better than `%v` when displaying the error. Can you please check how the messages look with `%v` vs. `%w`, and if `%w` looks the same/better, please switch all `fmt.Errorf(...)` uses to that